src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,8 @@
+package course_project;
+
+public class Main {
+
+    public static void main(String[] args) {
+        new SeaBattle().playVsComputer();
```

Comment:
Opt: Quite difficult to play, the field does not show placed ships, so I couldn't even start. Introduce random ship placement, if possible